app/src/main/res/layout/tv_shows_fragment.xml

```diff
@@ -1,13 +1,32 @@
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.coordinatorlayout.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     tools:context="ru.androidschool.intensiv.ui.tvshows.TvShowsFragment">
 
-    <!-- TODO: Update blank fragment layout -->
-    <TextView
+    <androidx.core.widget.NestedScrollView
```

Comment:
Тут лишний NestedScrollView, у нас же и так есть RecyclerView который будет скролится с сериалами. Лишний контейнер. Кроме этого когда RecyclerView вложен в ScrollView - это с точки зрения performance не хорошо. Если внутри recyclerView с match_parent по высоте в таком же ScrollView (c match_parent) - то смысл от RecyclerView теряется и все ячейки инициализируется сразу под количество элементов. В случае с 10 объектами (например 10 фильмов) это будет не заметно, но если объектов много и они тяжелые - то можно вылететь с OutOfMemory или при старте экрана будет тормозить пока все не проинициализируется.
Поэтому здесь достаточно одного виджета RecyclerView (даже без корневого лэйаута - чтобы минимальная вложенность была)

Reply:
> It's generally not a good idea to do this because you lose the main advantage of RecyclerView which is recycling. When RV is wrapped in NSV, it has to load everything it holds, even things that aren't visible.

https://www.reddit.com/r/androiddev/comments/1031p9c/comment/j2wbs44/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button

Reply:
+